zrml/prediction-markets/src/lib.rs

```diff
@@ -122,10 +121,11 @@ mod pallet {
             // TODO(#618): Not implemented for Rikiddo!
             T::DestroyOrigin::ensure_origin(origin)?;
 
-            let market = T::MarketCommons::market(&market_id)?;
+            let market =
+                <zrml_market_commons::Pallet<T> as MarketCommonsPalletApi>::market(&market_id)?;
```

Comment:
You can globally replace `<zrml_market_commons::Pallet<T> as MarketCommonsPalletApi>` by `<zrml_market_commons::Pallet<T>>` to shorten things up here.

```suggestion
            let market = <zrml_market_commons::Pallet<T>>::market(&market_id)?;
```

Reply:
I wonder why clippy don't suggest that :thinking:

Reply:
I wonder why I suggested it. My bad, of course we don't need the fully qualified syntax if there's no collisions between the function names.